src/PKSim.Presentation/Presenters/Simulations/PopulationPKAnalysisPresenter.cs

```diff
@@ -30,18 +30,16 @@ public class PopulationPKAnalysisPresenter : PKAnalysisPresenter<IPopulationPKAn
       private readonly List<PopulationPKAnalysis> _allPKAnalysesIndividualPKValues = new List<PopulationPKAnalysis>();
       private IPopulationDataCollector _populationDataCollector;
       private readonly IPopulationPKAnalysisToPKAnalysisDTOMapper _populationPKAnalysisToDTOMapper;
-      private readonly IGlobalPKAnalysisPresenter _globalPKAnalysisPresenter;
 
       public PopulationPKAnalysisPresenter(IPopulationPKAnalysisView view, IPKAnalysesTask pkAnalysesTask,
          IPKAnalysisExportTask exportTask, IPopulationPKAnalysisToPKAnalysisDTOMapper populationPKAnalysisToDTOMapper,
          IPKParameterRepository pkParameterRepository, IPresentationSettingsTask presentationSettingsTask,
          IGlobalPKAnalysisPresenter globalPKAnalysisPresenter)
-         : base(view, pkParameterRepository, presentationSettingsTask)
+         : base(view, pkParameterRepository, presentationSettingsTask, globalPKAnalysisPresenter)
       {
          _pkAnalysesTask = pkAnalysesTask;
          _exportTask = exportTask;
          _populationPKAnalysisToDTOMapper = populationPKAnalysisToDTOMapper;
-         _globalPKAnalysisPresenter = globalPKAnalysisPresenter;
          AddSubPresenters(_globalPKAnalysisPresenter);
```

Comment:
hum...this should go in the parent presenter and be gone here I think

Reply:
Oh, I see what you mean, the other lines referencing that object can go up too

Reply:
Fixed